src/plugins/usage_collection/README.md

```diff
@@ -196,11 +196,28 @@ In the case of using a custom ES or SavedObjects client, it is up to the plugin
 
 ##### Schema Field
 
-The `schema` field is a proscribed data model assists with detecting changes in usage collector payloads. To define the collector schema add a schema field that specifies every possible field reported (including optional fields) when registering the collector. Whenever the `schema` field is set or changed please run `node scripts/telemetry_check.js --fix` to update the stored schema json files.
+The `schema` field is a proscribed data model assists with detecting changes in usage collector payloads. To define the collector schema add a schema field that specifies every possible field reported (including optional fields) when registering the collector. The schema supports descriptions as simple strings that allow developers to document what the data represents. The `_meta` field only supports a description property.
+```
+schema: {
+  my_greeting: {
+    type: 'keyword',
+    _meta: {
+      description: 'The greeting keyword',
+    }
+  },
+  some_obj: {
+    total: {
+      type: 'long',
```

Comment:
I didn't add the `_meta` field here to show that it's optional. When we eventually move to descriptions being required, we'll need to update the example.

Reply:
NIT: Should we keep the example with the `_meta` field only? This way we indirectly encourage to provide them, by not making it that clear that it's optional. What do you think?

Reply:
Sounds good to me!